lib/extract/netcdf/Makefile

```diff
@@ -67,11 +67,11 @@ extract_netcdf_base.o: psy_data_base.o
 # Jinja rules for creating the base class, and the
```

Comment:
I'm unable to build the version of dl_esm_inf distributed with PSyclone with either Intel 17 or 19. I get:

    PSyclone/external/dl_esm_inf/finite_difference/src//field_mod.f90(1062): error #6638: An actual argument is an expression or constant; this is not valid since the associated dummy argument has the explicit INTENT(OUT) or INTENT(INOUT) attribute.   [B2]
    call exchange_generic(b2=self%get_data(), handle=exch, &
^

However, I can compile HEAD of master of dl_esm_inf OK. Does the submodule need updating?

Reply:
I've updated the submodule (I tested the intel compiler on a machine with no outside internet access ... so in short I forgot to copy a recent dl_esm version across). Done now, and it did now compile with Intel 19.